commit e14f4a1dd2b79f9bb7cebe028771363f45a39fc3 (refs/changes/19/2419/2)
Author: Cody Peter Mello <cody.mello@joyent.com>
Date:   2017-08-17T17:22:06+00:00 (2 years, 2 months ago)
    
    MANATEE-351 Update to pg_repack v1.4.1
    Reviewed by: Sean Chittenden <sean.chittenden@joyent.com>
    Reviewed by: David Pacheco <dap@joyent.com>
    Approved by: David Pacheco <dap@joyent.com>

diff --git a/deps/pg_repack b/deps/pg_repack
index cac8179..20f5bf4 160000
--- a/deps/pg_repack
+++ b/deps/pg_repack
@@ -1 +1 @@
-Subproject commit cac817929901796e1ed18f9592694a355464d9b3
+Subproject commit 20f5bf4878317143963f6cbeadbe520b4ef4b195
